src/service/standard-actions-impl.js

```diff
@@ -278,24 +278,27 @@ export class StandardActions {
      *
      *  Example: <button on="tap:AMP.copy(text='Hello AMP');copy-success:copied.show()">Copy</button>
      */
-    let eventName = CopyEvents.COPY_ERROR;
     if (isCopyingToClipboardSupported(win.document)) {
       copyTextToClipboard(
         win,
         textToCopy,
         () => {
-          eventName = CopyEvents.COPY_SUCCESS;
-          eventResult = 'success';
-          triggerEvent(eventName, eventResult, invocation);
+          triggerEvent(CopyEvents.COPY_SUCCESS, 'success', invocation);
         },
         () => {
-          eventResult = 'error';
-          triggerEvent(eventName, eventResult, invocation);
+          /**
+           * Trigger `COPY_ERROR` event with 'error' event result
+           * to indicate error while copying.
+           */
```

Comment:
See above re: JSDoc-style comments. Since the first ~line of this is describing the following line of code, it may not be adding any new information.
```suggestion
          // Error encountered while copying.
```